packages/packages/editor/src/init.tsx

```diff
@@ -1,6 +1,9 @@
 import ReactDOM from 'react-dom';
 import Shell from './components/shell';
+import { dispatchOnV1Ready } from '@elementor/v1-adapters';
 
 export default function init( domElement: HTMLElement ): void {
+	dispatchOnV1Ready();
```

Comment:
Consider: init instead of dispatch?

Reply:
thought about it, but `dispatch` is what it actually does, WDYT?